src/sagemaker/session.py

```diff
@@ -750,6 +750,26 @@ def create_endpoint(self, endpoint_name, config_name, wait=True):
             self.wait_for_endpoint(endpoint_name)
         return endpoint_name
 
+    def update_endpoint(self, endpoint_name, config_name):
```

Comment:
I didn't catch this earlier either.

Can we rename config_name parameter to endpoint-config-name, so that we are consistent with https://docs.aws.amazon.com/cli/latest/reference/sagemaker/update-endpoint.html